demo/src/main/java/com/novoda/merlin/demo/presentation/RxJava2DemoActivity.java

```diff
@@ -27,8 +27,8 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main);
         viewToAttachDisplayerTo = findViewById(R.id.displayerAttachableView);
-        networkStatusDisplayer = new NetworkStatusDisplayer(getResources(), merlinsBeard);
         merlinsBeard = MerlinsBeard.from(this);
+        networkStatusDisplayer = new NetworkStatusDisplayer(getResources(), merlinsBeard);
```

Comment:
@frapontillo I'd still rather not have this annotation. If I was to add it I would add it everywhere at the sametime.